bogota-july-2018/cyclomatic-complexity/Complejidad/kata.js

```diff
@@ -0,0 +1,10 @@
+(function (){
+
+  function predictAge(...ages){
+     let predict = ages.map( a => Math.pow(a,2));
+     return Math.floor(Math.sqrt(predict.reduce((a,b) => a+b))/2);
```

Comment:
Avoid train wreck. I Suggest  to move prediction to a variable.

`
let predictionReduction = predict.reduce((a,b) => a+b);
let prediction = Math.sqrt(predictionReduction / 2)
return Math.floor(prediction);
`